doc/rtd/topics/cli.rst

```diff
@@ -312,33 +349,66 @@ the currently running modules, as well as when it is done.
 
 .. code-block:: shell-session
 
-  $ cloud-init status
-  status: running
-
-  $ cloud-init status --long
-  status: running
-  time: Fri, 26 Jan 2018 21:39:43 +0000
-  detail:
-  Running in stage: init-local
-
-  $ cloud-init status
-  status: done
-
-  $ cloud-init status --long
-  status: done
-  boot_status_code: enabled-by-generator
-  last_update: Tue, 16 Aug 2022 19:12:58 +0000
-  detail:
-  DataSourceNoCloud [seed=/var/lib/cloud/seed/nocloud-net][dsmode=net]
-
-  $ cloud-init status --format=json
-  {
-   "boot_status_code": "enabled-by-generator",
-   "datasource": "nocloud",
-   "detail": "DataSourceNoCloud [seed=/var/lib/cloud/seed/nocloud-net][dsmode=net]",
-   "errors": [],
-   "last_update": "Tue, 16 Aug 2022 19:12:58 +0000",
-   "status": "done"
-  }
+   $ cloud-init status
```

Comment:
Maybe we just add a line of text before the command codeblocks? I'm open to suggestions. The highlighting does make it nicer to visually parse.

Reply:
I think the reason it looks odd is because there's no context around the commands, so you have the previous output immediately followed by the next command. According to my spreadsheet this page will go under "Explanation", so we can go one of two ways with it (depending on how users would be most likely to want to use the content here). 

(1) We can add context to explain each command before presenting it. In an explanation page we have the leeway to do that in a way that we probably wouldn't in e.g. a how-to, because it's a "learning oriented" page. In this way, if users want to copy the commands without the output examples, they can. ++ convenience.

(2) We can make the assumption that someone who is reading an "explanation" page is doing so from the POV of learning, and that they won't necessarily want to copy each command like they would in a tutorial or how-to. In this case, we can revert the change so it's a single code block again. Unfortunately this would also revert the changes to the highlighting (which I agree is nicer to look at). 

Personally, I would tend towards (1), because it would be more convenient for the users. I'm happy to add the text needed to break up the blocks and make the page look nicer, but equally happy to take advice if this isn't the preferred option.